src/pages/concept-list-page/concept-list-page-pure.tsx

```diff
@@ -14,37 +14,27 @@ interface Props {
   catalogId: string;
 }
 
-const createNewConceptAndNavigate = ({ history, catalogId }) => {
-  const body = {
-    anbefaltTerm: '',
-    status: 'utkast',
-    ansvarligVirksomhet: {
-      id: catalogId
-    }
-  };
+const createConcept = ({ catalogId }) => ({
+  anbefaltTerm: '',
+  status: 'utkast',
+  ansvarligVirksomhet: {
+    id: catalogId
+  }
+});
 
-  postConcept(body).then(response => {
-    // Get conceptId from response
-    const location = _.get(response, 'headers').get('Location');
-    const conceptId = location.split('/').pop();
+const createNewConceptAndNavigate = ({ history, catalogId }) =>
```

Comment:
history is a browser native API. Is it necessary to import it?

Reply:
aha, det er det ikke. jeg tror vi menere react-router history her! men skulle sjekke